webhook-server.go

```diff
@@ -287,7 +292,52 @@ func runScript() {
 	if err != nil {
 		log.Printf("err waiting for cmd: %s\n", err)
 	}
-	log.Print("command ran\n")
+
+	log.Printf("creating cloudfront invalidation\n")
+
+	_, awsIDPresent := os.LookupEnv("AWS_ACCESS_KEY_ID")
+	_, awsKeyPresent := os.LookupEnv("AWS_SECRET_ACCESS_KEY")
+
+	if awsIDPresent && awsKeyPresent {
+		svc := cloudfront.New(session.New())
+		bookID := getEnvOr("BOOK_CDN_DIST_ID", "")
+		docsID := getEnvOr("DOCS_CDN_DIST_ID", "")
+
+		if bookID != "" {
+			if err := invalidate(svc, bookID); err != nil {
+				log.Printf("error invalidating book cdn: %v\n", err)
+			}
+		}
+
+		if docsID != "" {
+			if err := invalidate(svc, docsID); err != nil {
+				log.Printf("error invalidating docs cdn: %v\n", err)
+			}
+		}
+	}
+
+	log.Printf("command ran\n")
```

Comment:
Can we log to a syslog facility please? And could we log json for easy log processing and searching? =)